src/App/public-components/Navbar/index.tsx

```diff
@@ -22,25 +32,48 @@ function Navbar(props: Props) {
     return (
         <nav className={_cs(className, styles.navbar)}>
             <div className={styles.appBrand}>
-                {process.env.MY_APP}
+                DEEP
             </div>
             <div className={styles.main}>
                 <div className={styles.navLinks}>
                     <SmartNavLink
                         exact
+                        route={route.home}
                         className={styles.link}
-                        activeClassName={styles.active}
-                        route={route.dashboard}
                     />
                     <SmartNavLink
                         exact
+                        route={route.tagging}
                         className={styles.link}
-                        activeClassName={styles.active}
-                        route={route.about}
                     />
+                    <SmartNavLink
+                        exact
+                        route={route.analysis}
+                        className={styles.link}
+                    />
+                </div>
+                <div className={styles.actions}>
+                    <ButtonLikeLink
+                        to={route.explore.path}
+                        variant="tertiary"
+                        icons={<IoCompassOutline />}
+                    >
+                        Explore DEEP
+                    </ButtonLikeLink>
+                    <QuickActionLink
+                        to="https://deephelp.zendesk.com/hc/en-us"
+                    >
+                        <IoHelp />
+                    </QuickActionLink>
+                    <QuickActionDropdownMenu
+                        label={<IoNotificationsOutline />}
+                        popupContentClassName={styles.notificationContent}
+                    >
+                        You don&apos;t have any notification
```

Comment:
Should we do this instead?
```suggestion
                        {'You don't have any notification'}
```